cms/djangoapps/cms_user_tasks/signals.py

```diff
@@ -46,8 +62,9 @@ def user_task_stopped_handler(sender, **kwargs):  # pylint: disable=unused-argum
 
         user_email = status.user.email
         task_name = status.name.lower()
+        olx_validation_text = get_olx_validation_from_artifact()
+        task_args = [task_name, str(status.state_text), user_email, detail_url, olx_validation_text]
         try:
-            # Need to str state_text here because it is a proxy object and won't serialize correctly
-            send_task_complete_email.delay(task_name, str(status.state_text), user_email, detail_url)
+            send_task_complete_email.delay(*task_args)
```

Comment:
were we not planning on having a separate email flag as well?

Reply:
Email is always sent when the import task is completed. so we decided not to have a flag around the email but instead `bypassing_olx_validations` which would add validation information in the email body conditionally.